urdf_model/include/urdf_model/pose.h

```diff
@@ -37,10 +37,18 @@
 #ifndef URDF_INTERFACE_POSE_H
 #define URDF_INTERFACE_POSE_H
 
+//For using the M_PI macro in visual studio it 
+//is necessary to define _USE_MATH_DEFINES
+#ifdef _MSC_VER
+#ifndef _USE_MATH_DEFINE
```

Comment:
Definitly. Clearly it was also working with a wrong `#ifndef` clause, and this is the reason why I did not notice it.